### PR TITLE
Explanation of 'changes on disk' dialog added, focusing on 'file comparison'

### PR DESCRIPTION
**Pull request by tknippenberg**

### CODE REVIEW

**Comment by tknippenberg** [comment]
@blis-mendix can you confirm whether I correctly described the functionality, specifically wrt the 'Modified' status?

**Comment by blis-mendix** [comment]
Yeah seems good

File: content/en/docs/refguide/installation/system-requirements.md [hunk 1]
```diff
@@ -109,6 +109,18 @@ To run a Mendix app, Mendix Studio Pro uses the following ports by default. If y
 
 For more information on ports and modifying Studio Pro's default ports, see [Configurations](/refguide/configuration/) and the [Troubleshooting Common Mobile Issues](/refguide/mobile/getting-started-with-mobile/prerequisites/#troubleshooting) section of *Native App Prerequisites and Troubleshooting*.
 
+### File Comparison
+
+{{% alert color="info" %}}
+Comparing files on disk with the original is currently not supported on macOS.
+{{% /alert %}}
+
+By default, Studio Pro points to an executable shipped together with TortoiseGit to compare files on disk. To customize this, navigate to **Preferences** > **Version control** > **General** > **File comparison** > **Executable** and choose your preferred tool.
```
Comment (quinntracy) [inline content/en/docs/refguide/installation/system-requirements.md]: @tknippenberg Could this first sentence be rephrased to "By default, Studio Pro uses an executable bundled with TortoiseGit for file comparison" or "By default, Studio Pro uses an executable bundled with TortoiseGit to compare files on disk"? Checking to ensure this wording wouldn't change the meaning.

Comment (tknippenberg) [reply content/en/docs/refguide/installation/system-requirements.md]: Thanks for checking. It's quite complicated haha.

There are two things:
* SP on a new device sets a default path to that executable. It's not checked whether TortoiseGit is installed, we just assume so.
* when installing TortoiseGit it by default installs to that path.

I think there is a subtle difference between "SP uses.." and "SP points to..", but if you think it's clear I'm fine either way!

**Comment by quinntracy** [comment]
@MariaShaposhnikova This can be merged after review/approval

File: content/en/docs/refguide/installation/system-requirements.md (hunk 1 above)
Comment (quinntracy) [reply content/en/docs/refguide/installation/system-requirements.md]: Thanks for providing additional details and helping clarify! I'll adjust appropriately using this feedback.

**Pull request merged**